lib/source.c

```diff
@@ -28,6 +28,8 @@
 #include <tlog/rc.h>
 #include <tlog/misc.h>
 #include <tlog/source.h>
+#include <tlog/timespec.h>
+#include <tlog/delay.h>
```

Comment:
Could you please wrap these includes in `#ifndef NDEBUG`? We don't use them otherwise here.